salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -64,6 +64,7 @@ Check node:
         # Sees: https://github.com/saltstack/salt/issues/28503
         service_cidr: {{ pillar.networks.service }}
         roles: {{ pillar.metalk8s.nodes[node_name].roles }}
+        folder: "/var/lib/containerd"
```

Comment:
Hmm to me you should keep the default from the module do not need to override it here